orchestrator/signer.go

```diff
@@ -168,43 +118,23 @@ func (l *ethSignerLoop) signBatch(ctx context.Context, batch *types.OutgoingTxBa
 	return nil
 }
 
-func (l *ethSignerLoop) getUnsignedValsets(ctx context.Context) ([]*types.Valset, error) {
-	var oldestUnsignedValsets []*types.Valset
-	getOldestUnsignedValsetsFn := func() (err error) {
-		oldestUnsignedValsets, err = l.inj.OldestUnsignedValsets(ctx)
-		if errors.Is(err, cosmos.ErrNotFound) || oldestUnsignedValsets == nil {
-			return nil
-		}
```

Comment:
same here @dbrajovic

Reply:
Same [explanation](https://github.com/InjectiveLabs/peggo/pull/106#discussion_r1497398563) for this as well